packages/sdk/src/utils/libp2p.ts

```diff
@@ -81,6 +81,10 @@ export async function defaultLibp2p(
 export async function createLibp2pAndUpdateOptions(
   options: CreateWakuNodeOptions
 ): Promise<Libp2p> {
+  if (options.contentTopics) {
+    options.shardInfo = { contentTopics: options.contentTopics };
```

Comment:
if we're overriding `shardInfo`, should there be some sort of a check that only one of `options.contentTopics` or `options.shardInfo` is set?

Reply:
Good point, I am still not sure the best way to tackle it.

Right now similar case is `shardInfo` and `pubsbuTopics` - where we prefer `shardInfo`. Same here for `contentTopics` - we prefer it over `shardInfo`. 

Main point is to have this check I think https://github.com/waku-org/js-waku/blob/cf24e60dd1257106f374f626800e0c86fda82818/packages/sdk/src/light-node/index.ts#L19

Reply:
Delaying removal of `pubsubTopics` till next release I suppose. 
This check is removed and addressing this point with additional logging - https://github.com/waku-org/js-waku/blob/77a85dfd35c994bee0e038ff5ecd0709292082ff/packages/sdk/src/utils/libp2p.ts#L129